eZ/Publish/Core/MVC/Symfony/Routing/Generator/UrlAliasGenerator.php

```diff
@@ -143,7 +143,7 @@ public function getPathPrefixByRootLocationId($rootLocationId, $languages = null
     public function isUriPrefixExcluded($uri)
     {
         foreach ($this->excludedUriPrefixes as $excludedPrefix) {
-            $excludedPrefix = '/' . trim($excludedPrefix, '/');
+            $excludedPrefix = '/' . ltrim($excludedPrefix, '/');
```

Comment:
Quite interesting given there's `eZ\Publish\Core\MVC\Symfony\Routing\Tests\UrlAliasGeneratorTest::testIsPrefixExcluded` coverage. That test definitely needs to cover the case which has proven to be error-prone.

Reply:
@alongosz : I could change the testIsPrefixExcluded - should I do this ?

I don't think that CI failing is related - have seen similar errors in other PRs - maybe @mnocon has an idea?

Reply:
> @alongosz : I could change the testIsPrefixExcluded - should I do this ?
> 

Yes please. Every bug-fix should have some coverage.

> I don't think that CI failing is related - have seen similar errors in other PRs - maybe @mnocon has an idea?

Yeah, seems global.

Reply:
In case you're still interested 😉 , it was the same issue as in https://github.com/ezsystems/ezplatform-kernel/pull/355#issuecomment-1369644048